src/app/core/common-components/entity-select/entity-select.component.spec.ts

```diff
@@ -65,6 +72,10 @@ describe("EntitySelectComponent", () => {
     component.entityType = User.ENTITY_TYPE;
     tick();
     fixture.detectChanges();
+    console.log(
+      "should suggest ... filteredEntities",
+      component.filteredEntities,
+    );
```

Comment:
```suggestion
```

Reply:
@christophscheuing can you remove the other debugging console outputs in the tests (or turn them into "expect()" statements?).

Reply:
I removed those debugging logs.

Reply:
I removed those logs.